src/Queue/src/Interceptor/Consume/Handler.php

```diff
@@ -19,14 +19,12 @@
  */
 final class Handler
 {
-    private readonly TracerFactoryInterface $tracerFactory;
     private readonly bool $isLegacy;
 
     public function __construct(
         private readonly HandlerInterface|CoreInterface $core,
-        ?TracerFactoryInterface $tracerFactory = null,
+        private readonly TracerFactoryInterface $tracerFactory = new NullTracerFactory(new Container()),
```

Comment:
@roxblnfk do you think it is bc break, if yes, I can skip `NewInInitializerRector`

Reply:
I think it's better to disable it here and apply it for v4 in the future.

Reply:
sure, I revered and skip `NewInInitializerRector` https://github.com/spiral/framework/pull/1141/commits/9cf5a48640591b4f997008b02789ac8c59ed15bd 👍